.sync.yml

```diff
@@ -1,6 +1,12 @@
 ---
 Rakefile:
   unmanaged: true
+Gemfile:
+  optional:
+    ':system_tests':
+      - gem: beaker-testmode_switcher
+        version: '<= 0.2.0'
```

Comment:
This looks like an accident - I think we need the `0.4.0` release for `execute_manifest_on` here and in Gemfile